manifests/concourse-manifest/concourse-base.yml

```diff
@@ -47,6 +51,19 @@ instance_groups:
         release: bpm
         properties: {}
 
+      - name: awslogs-xenial
```

Comment:
This is only one instance group, can we make this a [BOSH deployment manifest addon](https://bosh.io/docs/manifest-v2/#addons) instead so we also get logs from the workers